### PR TITLE
BIZ UDPMincho: Version 1.05 added; BIZ UDMincho: Version 1.05 added

### PR DESCRIPTION
**Pull request by aaronbell**
c27771f: [gftools-packager] BIZ UDMincho: Version 1.05 added

* BIZ UDMincho Version 1.05 taken from the upstream repo https://github.com/googlefonts/morisawa-biz-ud-mincho at commit https://github.com/googlefonts/morisawa-biz-ud-mincho/commit/7a76a4cc15a6d30b55aa6cdcb279e092796d756a.

 73a56b9: [gftools-packager] BIZ UDPMincho: Version 1.05 added

* BIZ UDPMincho Version 1.05 taken from the upstream repo https://github.com/googlefonts/morisawa-biz-ud-mincho at commit https://github.com/googlefonts/morisawa-biz-ud-mincho/commit/7a76a4cc15a6d30b55aa6cdcb279e092796d756a.

### CODE REVIEW

**Comment by aaronbell** [comment]
This update adds extends the Latin character set of the fonts, and also corrects an incompatibility by replacing the backspace glyph (formerly a Yen) to be a proper backspace per the Unicode standard.

**Comment by RosaWagner** [comment]
Thanks @aaronbell did you upgrade gftools before making the PR?

**Comment by aaronbell** [comment]
@RosaWagner oh I hadn’t. Things have changed again?

**Comment by RosaWagner** [comment]
Now the lang tags are not part supposed to be part of the metadata file anymore…

File: ofl/bizudmincho/METADATA.pb [hunk 1]
```diff
@@ -20,5 +20,220 @@ subsets: "latin-ext"
 subsets: "menu"
```
Comment (RosaWagner) [inline ofl/bizudmincho/METADATA.pb:20]: I don't see these subsets:
subsets: "chinese-simplified"
subsets: "chinese-traditional"
@aaronbell is it normal?

**Pull request merged**